runtime/autoload/netrw.vim

```diff
@@ -283,7 +283,7 @@ if !exists("g:netrw_scp_cmd")
  if executable("scp")
   call s:NetrwInit("g:netrw_scp_cmd" , "scp -q")
  elseif executable("pscp")
-  if (has("win32") || has("win95") || has("win64") || has("win16")) && filereadable('c:\private.ppk')
+  if (has("win32") || has("win64")) && filereadable('c:\private.ppk')
```

Comment:
Actually, this is still redundant.
`has("win32")` returns 1 on both 32-bit Windows and 64-bit Windows.
So, this can be:
```suggestion
  if has("win32") && filereadable('c:\private.ppk')
```